src/plugins/controls/public/options_list/components/options_list_popover.tsx

```diff
@@ -6,286 +6,67 @@
+import { OptionsListPopoverActionBar } from './options_list_popover_action_bar';
+import { OptionsListPopoverSuggestions } from './options_list_popover_suggestions';
+import { OptionsListPopoverInvalidSelections } from './options_list_popover_invalid_selections';
 
 export interface OptionsListPopoverProps {
   width: number;
   updateSearchString: (newSearchString: string) => void;
 }
 
-const aggregationToggleButtons = [
-  {
-    id: 'optionsList__includeResults',
-    label: OptionsListStrings.popover.getIncludeLabel(),
-  },
-  {
-    id: 'optionsList__excludeResults',
-    label: OptionsListStrings.popover.getExcludeLabel(),
-  },
-];
-
 export const OptionsListPopover = ({ width, updateSearchString }: OptionsListPopoverProps) => {
   // Redux embeddable container Context
-  const {
-    useEmbeddableDispatch,
-    useEmbeddableSelector: select,
-    actions: { selectOption, deselectOption, clearSelections, replaceSelection, setExclude },
-  } = useReduxEmbeddableContext<OptionsListReduxState, typeof optionsListReducers>();
-
-  const dispatch = useEmbeddableDispatch();
+  const { useEmbeddableSelector: select } = useReduxEmbeddableContext<
+    OptionsListReduxState,
+    typeof optionsListReducers
+  >();
 
   // Select current state from Redux using multiple selectors to avoid rerenders.
+  const existsSelectionInvalid = select((state) => state.componentState.existsSelectionInvalid);
   const invalidSelections = select((state) => state.componentState.invalidSelections);
-  const totalCardinality = select((state) => state.componentState.totalCardinality);
   const availableOptions = select((state) => state.componentState.availableOptions);
-  const searchString = select((state) => state.componentState.searchString);
   const field = select((state) => state.componentState.field);
 
-  const selectedOptions = select((state) => state.explicitInput.selectedOptions);
+  const existsSelected = select((state) => state.explicitInput.existsSelected);
   const hideExclude = select((state) => state.explicitInput.hideExclude);
-  const singleSelect = select((state) => state.explicitInput.singleSelect);
+  const fieldName = select((state) => state.explicitInput.fieldName);
   const title = select((state) => state.explicitInput.title);
-  const exclude = select((state) => state.explicitInput.exclude);
-
-  const loading = select((state) => state.output.loading);
-
-  // track selectedOptions and invalidSelections in sets for more efficient lookup
-  const selectedOptionsSet = useMemo(() => new Set<string>(selectedOptions), [selectedOptions]);
-  const invalidSelectionsSet = useMemo(
-    () => new Set<string>(invalidSelections),
-    [invalidSelections]
-  );
 
   const [showOnlySelected, setShowOnlySelected] = useState(false);
-  const euiBackgroundColor = useEuiBackgroundColor('subdued');
 
   return (
-    <>
+    <span role="listbox" aria-label={OptionsListStrings.popover.getAriaLabel(fieldName)}>
```

Comment:
This was a quick `a11y` fix.

- Before:
    
    ![image](https://user-images.githubusercontent.com/8698078/199792791-0c8ee189-1eaa-4c18-be30-3572460c593b.png)

- After:

     ![image](https://user-images.githubusercontent.com/8698078/199792888-05233f0b-e889-4e40-af82-2286ff672c61.png)